duckduckgo/macos-browser
# Add a debug menu option for sending test NetP notifications

Task/Issue URL: https://app.asana.com/0/1199230911884351/1204921820737754/f
Tech Design URL:
CC:

**Description**:

This PR adds a new Debug menu option called `Send Test Notification`. It tells the NetP extension to trigger a notification, making it easy to test this feature quickly, without having to disconnect your Internet connection and wait for the connectivity tester to trigger.

**Steps to test this PR**:
1. Bump the version of the app using `./set_version x.y.z`, to ensure that the extension gets updated
2. Open the Debug menu and choose "Send Test Notification"
3. Verify that a notification is presented, or possibly a prompt to allow notifications. If you get this prompt, then allow notifications and test the notification feature again.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
